PermissionsEx/permissions.yml

```diff
@@ -595,6 +595,35 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+ terminator:
+    options:
+      prefix: '&8T&7er&7m&8i&7n&4a&8t&7o&8r'
+      suffix: '&8'
+    inheritance:
+    - mortal
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.jump0
+        - ch.alias.carryplayer
+        - ch.alias.dismountplayer
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.run0
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.haste0
+        - ch.alias.haste1
+        - ch.alias.haste2
+        - ch.alias.haste3
+        - ch.alias.haste4
+        - ch.alias.haste5
+        - ch.alias.haste6
+        - ch.alias.haste7
+        - ch.alias.haste8
```

Comment:
https://video.search.yahoo.com/yhs/search?fr=yhs-omr-001&hsimp=yhs-001&hspart=omr&p=the+terminator+beating+up+bill+paxton#id=1&vid=bda2c7df659042b251ae8ef4a2445911&action=click